liquibase-standard/src/main/java/liquibase/sqlgenerator/core/CreateTableGenerator.java

```diff
@@ -255,8 +255,8 @@ public Sql[] generateSql(CreateTableStatement statement, Database database, SqlG
                 buffer.append(database.escapeColumnNameList(StringUtil.join(getPrimaryKeyColumns(statement.getPrimaryKeyConstraint().getColumns(), database, autoIncrementColumns), ", ")));
                 buffer.append(")");
                 // Setting up table space for PK's index if it exist
-                if (((database instanceof OracleDatabase) || (database instanceof PostgresDatabase)) && (statement
-                        .getPrimaryKeyConstraint().getTablespace() != null)) {
+                if (((database instanceof OracleDatabase) || (database instanceof PostgresDatabase)) && (StringUtil.trimToNull(statement
```

Comment:
@delayk  maybe a StringUtil.isNotEmpty would be easier to understand here?

Reply:
I found the usage of `StringUtil.trimToNull(property) != null` in some other places, such as `liquibase.change.core.CreateTableChange` line 156. I thought this was a standard Liquibase development guideline.
`StringUtil.isNotEmpty` is ok for me.

Reply:
I see. TrimToNull is really legacy code that has been around for ~15 years :) 
isNotEmpty is more 2015's .
Thanks!